include/net/nrf_cloud_codec.h

```diff
@@ -76,8 +75,30 @@ int nrf_cloud_service_info_json_encode(const struct nrf_cloud_svc_info * const s
  * @retval 0 If successful.
  * @return A negative value indicates an error.
  */
-int nrf_cloud_modem_info_json_encode(const struct nrf_cloud_modem_info * const mod_inf,
+int nrf_cloud_modem_info_json_encode(const struct nrf_cloud_modem_info *const mod_inf,
 				     cJSON * const mod_inf_obj);
```

Comment:
You can adjust the `cJSON * const` to be `cJSON *const` as well

Reply:
compliance complained about `*` spacing being inconsistent... so i made them consistent and it is happy now 🤷‍♂️

Reply:
Well, that's wild. I guess if compliance is okay with it, I'm okay with it